test/integration/node-middleware.test.ts

```diff
@@ -178,6 +178,8 @@ describe("createNodeMiddleware(webhooks)", () => {
     // @ts-expect-error complains about { port } although it's included in returned AddressInfo interface
     const { port } = server.address();
 
+    const payload = '{"name":"invalid"';
```

Comment:
Can you explain why the test needed to be updated?

Reply:
Before it was parsing and then validating. So in the test was an invalid sha256. So it would parse and fail on validation. 

I changed the sha256 to pass the validation but the payload will fail on parsing.